test/support/WireMockResponses.scala

```diff
@@ -240,8 +243,53 @@ object WireMockResponses {
 
   }
 
-  def customerDataOk = {
-    stubFor(get(urlMatching("/vat/customer/vrn/.*"))
+  def financialsOkSingle = {
+    stubFor(get(urlMatching("/enterprise/financial-data/VRN/.*"))
+      .willReturn(aResponse()
+        .withStatus(200)
+        .withBody(
+          s"""
+             {
```

Comment:
How about extracting these jsons to some `TestData` object and reuse themfor both wiremock stubs and json formats specs?

Reply:
Done